### PR TITLE
fixes to image()

### LINKED ISSUE #4280: Issue with "image()": (Providing fallback images section out of date)
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/image()

#### What information was incorrect, unhelpful, or incomplete?
Spec removed mime type fallbacks from the image() function in favor of image-set() function use

#### Specific section or headline?
Providing fallback images

#### What did you expect to see?
the entire `Providing fallback images` section should be removed

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/image()`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/image()
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/image()/index.html
* Last commit: https://github.com/mdn/content/commit/f4fb4c2d90a134366c280eec6cb599825981ad5d
* Document last modified: 2021-03-15T17:55:56.000Z

</details>

### PR DESCRIPTION
**Pull request by rachelandrew**
Fixes #4280 also fixable flaws and removing the experimental macro/.

### CODE REVIEW

**Comment by jsnkuhn** [comment]
the `Providing fallback images` headed section still needs to be removed

File: files/en-us/web/css/image()/index.html [hunk 1]
```diff
@@ -9,9 +9,9 @@
   - Reference
   - Web
 ---
-<div>{{CSSRef}}{{SeeCompatTable}}</div>
+<div>{{CSSRef}}</div>
 
-<p>The <strong><code>image()</code></strong> <a href="/en-US/docs/CSS">CSS</a> <a href="/en-US/docs/Web/CSS/CSS_Functions">function</a> defines an {{CSSxRef("&lt;image&gt;")}} in a similar fashion to the {{CSSxRef("&lt;url&gt;")}} function, but with added functionality including specifying the image's directionality, specifying fallback images for when the preferred image is not supported, displaying just a part of that image defined by a media fragment, and specifying a solid color as a fallback in case none of the specified images are able to be rendered.</p>
```
Comment (chrisdavidmills) [inline files/en-us/web/css/image()/index.html:14]: I'm not really sure why you deleted "specifying fallback images for when the preferred image is not supported" from this sentence — this seems like an obvious case to me, and in fact, the example you add below shows that use case.

**Review by chrisdavidmills — COMMENTED** [review]
Just one comment from me; thanks @rachelandrew !

File: files/en-us/web/css/image()/index.html (hunk 1 above)
Comment (rachelandrew) [reply files/en-us/web/css/image()/index.html:14]: @chrisdavidmills that is the thing we were removing in this PR, plus the example, so maybe I'm confused. The fallback color remains in the spec but not fallback images so I removed this text in the intro and the example below.

Comment (chrisdavidmills) [reply files/en-us/web/css/image()/index.html:14]: Oh, OK, sorry! Now I understand!

**Review by chrisdavidmills — APPROVED** [review]

**Pull request merged**